lib/MusicBrainz/Server/Controller/EditNote.pm

```diff
@@ -105,8 +105,8 @@ sub detach_if_cannot_change {
     }
 
     my $now = DateTime->now( time_zone => $edit_note->post_time->time_zone );
-    my $note_age = $now - $edit_note->post_time;
-    my $is_note_too_old = $note_age->{days} > 0;
+    my $note_age = $now->delta_days($edit_note->post_time);
+    my $is_note_too_old = $note_age->days > 0;
```

Comment:
Doesn’t it prevent changing an edit note posted at 23:59:59 after 1 second?

Reply:
If that's the case, then https://metacpan.org/pod/Date::Calc#3) should be useful.

The main issue with testing date differences is that it usually requires wrapping the Perl tests in a wrapper that sets a static date (we have one test that does that somewhere) but I guess we could do that :)